front/src/App.js

```diff
@@ -1,15 +1,26 @@
 import React from "react";
 import "./index.css";
-import Header from "./component/Header";
-import Main from "./component/Main";
+import AdminHeader from "./component/AdminHeader";
+import Admin from "./component/Admin";
+import {Route} from "react-router-dom";
+import Main from "./home/Main";
+import Footer from "./home/Footer";
+import MainHeader from "./home/MainHeader";
 
 const App = () => {
-  return (
-    <div className="App">
-      <Header />
-      <Main />
-    </div>
-  );
+    return (
+        <div>
+            <Route exact path="/">
+                <MainHeader/>
+                <Main/>
+            </Route>
+            <Route path="/admin">
+                <AdminHeader/>
+                <Admin/>
+            </Route>
+            <Footer/>
```

Comment:
저희 린트 설정상으로 열고 닫는게 아닌 바로 닫아도 되는 컴포넌트 (예를 들면 위의 <MainHeader />와 같은)들은 /하기 전에 공백하나를 넣는게 디폴트로 지정되어있다고 알고있습니다.

포매터를 한 번 돌려보면 좋을 거 같아요!

```suggestion
                <MainHeader />
                <Main />
            </Route>
            <Route path="/admin">
                <AdminHeader />
                <Admin />
            </Route>
            <Footer />
```